### ISSUE TITLE
Extra space is added

### ISSUE THREAD

**Issue by fersarr** [ShailChoksi/text2digits#15]
Hi, thanks for this package, it's very useful! Just wanted to report this weird behaviour:

I know negative numbers are not yet supported but it shouldn't move the spaces around or add extra ones:

```
>>> t2d.convert('hey this is a number -7')
'hey this is a number- 7'
```

I would expected the text to end up unmodified:

```
>>> t2d.convert('hey this is a number -7')
'hey this is a number -7'
```

The same happens with:
```
>>> t2d.convert('-7')
'- 7'
```

**Comment by ShailChoksi** [comment]
This should be fixed in version 0.0.6. Can you test and confirm?
```
>>> t2d.convert("hey this is a number -7")
'hey this is a number -7'
>>> t2d.convert("-7")
'-7'
>>> t2d.convert("- sevent")
'- sevent'
>>> t2d.convert("- seven")
'- 7'
>>> t2d.convert("-seven")
'-7'
```

**Comment by fersarr** [comment]
you are right! thanks :)
